packages/api/src/index.ts

```diff
@@ -1,10 +1,20 @@
+import { getHyperDXMetricReader } from '@hyperdx/node-opentelemetry/build/src/metrics';
+import { HostMetrics } from '@opentelemetry/host-metrics';
+import { MeterProvider } from '@opentelemetry/sdk-metrics';
 import { serializeError } from 'serialize-error';
 
 import * as config from '@/config';
 import Server from '@/server';
 import { isOperationalError } from '@/utils/errors';
 import logger from '@/utils/logger';
 
+// Start collecting host metrics
```

Comment:
sending host metrics to help debugging cpu/memory issue on hyperdx going forward